configs/default/MTKS-MATEKF411.config

```diff
@@ -3,7 +3,9 @@
 #mcu STM32F411
 
 #define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM20689
 #define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM20689
```

Comment:
Matek F411 does not have ICM20689

http://www.mateksys.com/?portfolio=f411-mini

Reply:
ok，looks like darwin  not have 4.4 config for their  babyape  v1